docs/administration.projects.md

```diff
@@ -79,21 +77,26 @@ This export is an extension of the [Project Export](#project-export). It adds th
 Sometimes additional information about the lifecycle of the project is required. The archive icon in the list of deleted projects shows all major events that happened on the project. This history is also available for existing projects in the **Customers** section of the **Administration** area. Navigate to the **Customer Projects** of a specific customer and click the archive icon on the project you want to see details about.
 
 The following events are available:
-
-- **PROJECT_CREATED**: A new meshProject was created.
-- **PROJECT_DELETED**: A meshProject was deleted by a customer admin.
-- **PROJECT_DELETION_CONFIRMED**: A partner or platform operator confirmed a meshProject deletion that required manual deletion of platform tenants.
-- **PROJECT_DELETION_DECLINED**: A partner or platform operator declined a meshProject deletion that required manual deletion of platform tenants.
-- **ALL_PROJECT_TENANTS_DELETED**: Tenants of a meshProject are deleted asynchronously after the user deleted the project. When this deletion is successful, this event is written.
-- **TENANT_ADDED**: One tenant was added to a project.
-- **TENANT_REMOVED**: One tenant was removed from a project.
-- **USER_ASSIGNED**: A user was assigned to the project. If [4-EP](meshstack.authorization.md#user-project-role-approval) is active, this event is written after successful approval of the role request.
-- **USER_UNASSIGNED**: A user was removed from the project.
-- **PROJECT_ROLE_CHANGED**: The user role was changed on the project. If [4-EP](meshstack.authorization.md#user-project-role-approval) is active, this event is written after successful approval of the role request.
-- **PROJECT_ROLE_APPROVED**: If [4-EP](meshstack.authorization.md#user-project-role-approval) is active, this event is written for every approval.
-- **PROJECT_ROLE_REJECTED**: If [4-EP](meshstack.authorization.md#user-project-role-approval) is active, this event is written when a role request is rejected.
-- **EDIT_PROJECT_TAGS**: If Project Tags are configured, this event is written every time a tag is edited by a customer admin.
-- **PARTNER_EDIT_PROJECT_TAGS**: If Project Tags are configured, this event is written every time a partner/admin edits the restricted project tags.
-- **QUOTA_EDITED**: Quota Settings for the project were changed.
-- **SERVICE_USER_CREATED**: A Service User was created for the project.
-- **SERVICE_USER_DELETED**: A Service User was deleted for the project.
+| Event | Description|
+| ------| -----------|
+| **PROJECT_CREATED**| A new meshProject was created |
+| **PROJECT_PROJECT_MARKED_FOR_DELETION**<br>`since v7.x.x.`| A meshProject was deleted by a customer admin.|
+|**PROJECT_DELETED**| `since v7.x.x` When the deletion of all tenants is successful, this event is written.<br>`until v7.x.x`A meshProject was deleted by a customer admin.|
```

Comment:
@uleMeshcloud here for example

Reply:
ah thanks. I will adapt it to make it more clear